src/main/java/org/opentripplanner/api/common/RoutingResource.java

```diff
@@ -151,7 +151,40 @@
     @QueryParam("optimize")
     protected OptimizeType optimize;
     
-    /** The set of modes that a user is willing to use, with qualifiers stating whether vehicles should be parked, rented, etc. */
+    /**
+     * <p>The set of modes that a user is willing to use, with qualifiers stating whether vehicles should be parked, rented, etc.</p>
+     * <p>The possible values of the comma-separated list are:</p>
+     *
+     * <ul>
+     *  <li>WALK</li>
+     *  <li>TRANSIT: General catch-all for all public transport modes.</li>
+     *  <li>BICYCLE: Taking a bicycle onto the public transport and cycling from the arrival station to the destination.</li>
```

Comment:
Transit isn't required with the BICYCLE mode. It's possible to send just BICYCLE and get a bike-only result.